src/repository.js

```diff
@@ -31,9 +31,13 @@ module.exports = class Repository {
   async insert(entityInstance) {
     const payload = this.dataMapper.collectionFieldsWithValue(entityInstance)
 
-    let { ops } = await this.runner().collection(this.collectionQualifiedName).insertOne(payload)
+    let ret = await this
```

Comment:
use `const` instead let, this variable don't receive a new value

Reply:
done